support/slice/helpers.go

```diff
@@ -0,0 +1,12 @@
+package slice
+
+// Check item in slice string type
+func Contains(slice []string, item string) bool {
+	set := make(map[string]struct{}, len(slice))
+	for _, s := range slice {
+		set[s] = struct{}{}
```

Comment:
I can't find this function used. Maybe you can remove this.

*ps: Any reason why using map. Why just not return if  `s == item` directly ?

Reply:
it is used internet_test.go - 24

Reply:
ok